bigbluebutton-tests/playwright/user/multiusers.js

```diff
@@ -219,6 +219,44 @@ class MultiUsers {
     await this.userPage.hasText(`:nth-match(${e.dropdownWebcamButton}, 2)`, this.userPage.username, 'should display the username of Attendee on the second pinned webcam for Attendee');
   }
 
+  async focusUnfocusWebcam() {
+    await this.modPage.shareWebcam();
+    await this.userPage2.shareWebcam();
+
+    await this.modPage.getLocator(e.dropdownWebcamButton)
+      .filter({ hasText: this.userPage2.username })
+      .click();
+
+    await this.modPage.wasRemoved(e.focusWebcamBtn, 'should not display the focus webcam button for the moderator');
```

Comment:
try to be more descriptive and contextualized in the description messages: this helps other devs to understand what is expected, why that is expected, and what step the test was. as this message is logged together with the other playwright error logs, the more precise and clear it is, the better it will be to spot the failure reason and reproduce it!

please also check if other messages could also have any details that would help

```suggestion
    await this.modPage.wasRemoved(e.focusWebcamBtn, 'should not display the webcam focus button to the moderator when there are less than 3 webcams sharing');
```